core/parentchain/block-import-dispatcher/src/immediate_dispatcher.rs

```diff
@@ -22,7 +22,7 @@ use std::{boxed::Box, vec::Vec};
 
 /// Block import dispatcher that immediately imports the blocks, without any processing or queueing.
 pub struct ImmediateDispatcher<BlockImporter> {
-	block_importer: BlockImporter,
+	pub block_importer: BlockImporter,
```

Comment:
I needed to make a few fields public in order to access the shard vault entry which is quite deeply nested.
the alternative would be to pass a getter through all hierarchies, but I'm not sure that's worth it. thoughts, @clangenb